src/ExadelMentorship.BusinessLogic/Features/WeatherFeature/FutureWeather/FutureWeatherCommandResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ExadelMentorship.BusinessLogic.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ExadelMentorship.BusinessLogic.Features.WeatherFeature.FutureWeather
+{
+    public class FutureWeatherCommandResponse
+    {
+        public List<City> cityList { get; set; }
+        public string Error{ get; set; }
```

Comment:
Can't see where this property being used. Either you are not handling error case, or we don't need this `FutureWeatherCommandResponse` class at all and use `ICommand<IEnumerable<City>>` instead.

P.S. Try to use `IEnumerable<>` where possible instead of `List<>` as, `IEnumerable` is a higher-level abstraction.

Reply:
done